teams/vsp/teams/tools/backend/triage/ongoing-support-developer.md

```diff
@@ -1,94 +1,139 @@
-* Supporting oncall develeoper
+* Requests for general assistance in the [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) channel
+* On-call developer support requests
 * Any internal VSP team PRs
+* Review Sentry errors
 * Small, non-critical issues that can be easily stopped/started
 
-There is no intention to permanently assign this role to a single person -- this is a shared responsibility assigned to the entire team much like handling oncall responsibilities -- and each developer on the team will take a turn on a rotating basis.
+## Responsibilities
+
+As issues come in, the main job is to: 
 
-## What’s the time commitment (Answer: the entire sprint)
+1. acknowledge the incoming request
+1. decide on the best person to handle the request
+1. follow up to make sure the request is completed in a timely manner.
 
-This task is part of several ongoing sprint goals:
+It is *100%* acceptable to delegate the request, whatever it is, to someone more suited to the task - but the first point of contact should be the utility developer. Unless it's an emergency, waiting until standup to discuss assignments for requests helps to minimize interruptions for everyone.
 
-* Triage https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/2297
-* Incident Response https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/1954
-* Global Customer Support https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/3675
+---
 
-Each developer will get assigned a sprint and will only be responsible for the objectives related to this ongoing responsibility for that sprint. Since this covers *three* ongoing SuperEpics, we’re assigning it like we’d assign any other sprint deliverable.
+**The utility developer will...**
 
-The utility developer assignment schedule is on PagerDuty: https://dsva.pagerduty.com/schedules#PPGUD8L
+* be assigned to a developer for an entire sprint
+* be generally available during our standard support/business hours: **M-F 9:00a - 5:00p ET**  :clock5: (adjustable based on time zone)
 
-The role will be assigned to a developer for an entire sprint, running during our standard support/business hours (Mon - Fri, ~9am - 5pm Eastern) starting on Wed at 11am on the first day of the sprint and handing off to the next developer two weeks later when the next sprint begins.
+### At the beginning of the sprint
 
+**The utility developer will...**
 
-## What do I do?
+* join the [#vsp-backend-utility-dev] Slack channel
+* make sure they are assigned to both of the support [Super Epics]
+* unmute [#vfs-sentry-alerts], [#oncall], and [#vfs-platform-support] Slack channels (if they're muted)
+* update the [#vfs-platform-support] channel topic with their name
+* attend a hand-off meeting which occurs at 11:00a ET on the planning day (Wednesda) before a new sprint
```

Comment:
Wednesday has a "y" at the end.  Also, do you want to change this to 1pm ET?  That way we can schedule the the hand off right after sprint planning?